src/pages/graphql/payment-methods/payment-services-paypal-vault.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Payment Services Paypal Vault payment method
+description: Learn how to use the GraphQL API mutation for the Payment Services Paypal Vault payment solution.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment Services Paypal Vault payment method
+
+Payment Services Paypal Vault is a payment method where a stored card is used for the transaction.
+
+The following conditions must be true to use this payment method:
+
+-  The shopper must be a logged-in customer.
```

Comment:
```suggestion
-  The customer must be logged-in.
```

I only use the word shopper in the GraphQL context when it's possible that the person can be a logged-in customer or a guest.